### PR TITLE
Add informations to cargo order form

### PR DESCRIPTION
**Pull request by corentt**
## About the PR 
Displays the product name, unit cost and description in the small cargo order form window so you actually see what you are ordering. Also adds a tooltip with the description of the product on the main order window.
It also fix the descriptions for ATV and syndie secway

**Screenshots**
**edit:** Updated screenshots
scrot
scrot2

**Changelog**
:cl:
- add: More detailed informations are now displayed on the cargo order form

### CODE REVIEW

File: Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs [hunk 1]
```diff
@@ -68,10 +69,20 @@ protected override void Open()
             {
                 if (args.Button.Parent is not CargoProductRow row)
                     return;
+
+                if (row.Description.Text != null)
+                {
+                    var Description = new RichTextLabel();
```
Comment (mirrorcult) [inline Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs:75]: this is a local variable which is going unused?

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl [hunk 2]
```diff
@@ -14,7 +14,7 @@ ent-CrateFunBoardGames = Board game crate
     .desc = Game nights have been proven to either decrease boredom or increase murderous rage depending on the game.
 
 ent-CrateFunATV = ATV crate
-    .desc = Bring some jazz to the station with the brass ensemble. Contains a variety of brass instruments for the whole station to play.
+    .desc = An Absolutely Taxable Vehicle to help cargo with hauling.
 
 ent-CrateFunSyndicateSegway = Syndicate segway crate
-    .desc = Bring some jazz to the station with the brass ensemble. Contains a variety of brass instruments for the whole station to play.
+    .desc = An All-Tile Vehicle to help security run after clowns
```
Comment (mirrorcult) [inline Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl:20]: the syndie segway isn't for security and isn't an ATV

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs (hunk 1 above)
Comment (corentt) [reply Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs:75]: You're right, I'll remove it.
I'm new to this project and had to try several ways before I got what I wanted, and I forgot to remove this line before commit.

File: Content.Client/Cargo/UI/CargoProductRow.xaml [hunk 3]
```diff
@@ -23,5 +23,9 @@
                    MinSize="52 32"
                    Align="Right" />
         </PanelContainer>
+        <Tooltip Name="Description"
+            Access="Public"
+            Visible="False"
+            Text=""/>
```
Comment (metalgearsloth) [inline Content.Client/Cargo/UI/CargoProductRow.xaml:29]: Unused

**Comment by corentt** [comment]
> The setsize no longer fits the entire control so should be expanded vertically. Also the other text is white but this is grey (should it be rich text?)

It is grey because this is default color of richtext and richtext is the only way i found to get the text on multiple lines. I will force it to white.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**